.prettierignore

```diff
@@ -2,3 +2,4 @@ node_modules
 pnpm-lock.yaml
 dist
 target
+/.github
```

Comment:
for some reason the linter was complaining about the workflow file.. but I don't think we should really be running on those anyway